src/microsim/output/MSE2Collector.cpp

```diff
@@ -1553,15 +1553,15 @@ double
 MSE2Collector::getEstimateQueueLength() const {
 
     if (myVehicleInfos.empty()) {
-        return -1;
+        return 0;
```

Comment:
Although this code part does not seem to be used much, it is an additional value to know immediately whether the queue length is due to absence of vehicles or they just didn't stop...